net/babeld/Makefile

```diff
@@ -14,7 +14,12 @@ PKG_RELEASE:=1
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://www.irif.fr/~jch/software/files/
 PKG_HASH:=98070dc418c190f047b8d69eb47987df30ded8f0fca353c49427d3137ad08b87
+
 PKG_LICENSE:=MIT
```

Comment:
PKG_MAINTAINER:= should go here. If none, leave it empty.

Reply:
Updated.